EntityModel/DbModel.cs

```diff
@@ -6,17 +6,23 @@ namespace EntityModel
 {
     public class DbModel : DbContext
     {
+        public DbSet<User> Users { get; set; }
         public DbSet<Organization> Organizations { get; set; }
-        public DbSet<Snapshot> Snapshots { get; set; }
+        public DbSet<Service> Services { get; set; }
+        public DbSet<CaseManagementData> CaseManagementData { get; set; }
+        public DbSet<HousingData> HousingData { get; set; }
+        public DbSet<JobTrainingData> JobTrainingData { get; set; }
+        public DbSet<MentalHealthData> MentalHealthData { get; set; }
+        public DbSet<SubstanceUseData> SubstanceUseData { get; set; }
```

Comment:
The models are shared between all API interfaces (Dynamics and Entity Framework). The layout here for EF reflect the setup in Dynamics so that the tests can use an in-memory DB to run the same code locally